product-service/src/functions/getProductsList/handler.ts

```diff
@@ -0,0 +1,39 @@
+import 'source-map-support/register';
+
+import { formatJSONResponse } from '@libs/apiGateway';
+import { middyfy } from '@libs/lambda';
+import { PG_getAllProducts } from '@app/services/pg_getAllProducts';
+import { FormatJSONResponseType, Sneaker } from '@app/types/types';
+import { STATUS } from '@app/constants';
+import { logger } from '@app/utils/logger';
+
+export const handler = async (): Promise<FormatJSONResponseType> => {
+  try {
+    const product: Sneaker[] = await PG_getAllProducts();
+
+    if (!product || product.length === 0) {
+      return formatJSONResponse({
+        statusCode: STATUS.SERV_ERR,
+        product: [],
+      });
+    }
+    logger.info({
```

Comment:
U should move this logging upper. If error happens U will never know where it occurs